main.go

```diff
@@ -34,18 +28,17 @@ func checkForErrorsAndExit(err error) {
 	if err == nil {
 		os.Exit(0)
 	} else {
-		logger := util.CreateLogger("")
 		if os.Getenv("TERRAGRUNT_DEBUG") != "" {
-			logger.Println(errors.PrintErrorWithStackTrace(err))
+			util.LogEntry.Debugf(errors.PrintErrorWithStackTrace(err))
 		} else {
 			// Log error in red so that it is highlighted
-			util.ColorLogf(logger, color.New(color.FgRed), err.Error())
+			util.LogEntry.Errorf(err.Error())
```

Comment:
I'd get rid of the if-else, and leave the `Debugf` and then `Errorf`. The right data will show up based on what log level you have set.

Reply:
re terragrunt version: I initially moved it to later stage. Now with global logger it makes sense to move it back.